ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Notification/SubscriptionsSample.cs

```diff
@@ -375,12 +375,12 @@ public void ShowAllTeamSubscriptions()
         }
 
         /// <summary>
-        /// Follow a work item (get notified about certain updates to a work item)
+        /// Follow a work item (get notified about certain updates to a work item),
+        /// and then unfollow the same workitem (stop getting notified about the certain updates to the above work item)
         /// </summary>
-        /// <param name="workItemId"></param>
         /// <returns></returns>
         [ClientSampleMethod]
-        public NotificationSubscription FollowWorkItem()
+        public NotificationSubscription FollowUnfollowWorkItem()
```

Comment:
You should have separate methods for follow and unfollow. Sample methods are called in order (when they get run via the "runner" tool) so you can have your follow method follow a work item, then have your unfollow method unfollow it.

And the best sample for "unfollow" probably shows how to start from a work item ID (not a subscription ID, since this isn't what most people walking up to the sample will know/have). I think this means you need to query for the follow subscription and delete it if found.

Reply:
Sure @willsmythe, created a separate sample for Unfollow.

Reply:
I will update the line numbers on the PR to Content.VS once I merge this.